wordpress-mobile/gutenberg-mobile
# Add post title to Gutenberg Mobile

## Description:

Implements the editor title in RN for Gutenberg, also removes the native title.

This PR also adds the `Noto` font family to the repository.

## Focus points:

For @hypest and anyone else wanting to chime in:

1. Please review if the way in which I'm storing the title in `AppContainer.js` makes sense.  I'm not sure how it would work to have that in redux so I just went for a simple initial solution.
2. My idea in this PR is to offer a working solution so we have a good baseline to work on.  While I do want to make this perfect, I also suggest that we try to be a bit optimistic with this and work incrementally on things that are not too urgent / serious.  Just a though since the PRs for this feature are already quite complicated.

## Related PRs:

gutenberg: https://github.com/WordPress/gutenberg/pull/13199
WPiOS: https://github.com/wordpress-mobile/WordPress-iOS/pull/10794
WPAndroid: https://github.com/wordpress-mobile/WordPress-Android/pull/9027

## Testing: 

### Test 1:

1. Run the app.
2. Edit the title and make sure the state is properly maintained.
3. Tap save.

Check the iOS console log that the title is properly retrieved. (not sure if there's a similar log in Android)

### Test 2:

1. Run the app.
2. Delete the title and make sure the placeholder is shown.
